README.md

```diff
@@ -37,7 +37,6 @@ meminfo | Exposes memory statistics. | Darwin, Dragonfly, FreeBSD, Linux
 netdev | Exposes network interface statistics such as bytes transferred. | Darwin, Dragonfly, FreeBSD, Linux, OpenBSD
 netstat | Exposes network statistics from `/proc/net/netstat`. This is the same information as `netstat -s`. | Linux
 sockstat | Exposes various statistics from `/proc/net/sockstat`. | Linux
-stat | Exposes various statistics from `/proc/stat`. This includes CPU usage, boot time, forks and interrupts. | Linux
```

Comment:
Hrm good point.. I think best would be to keep non-cpu stuff in the stat collector, if we can do that without much complication/code duplication.
@rtreffer What do you think?

This also reminds me of @derekmarcotte's refactoring suggestions which would be helpful here. But let's keep things simple for now.